core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
@sellout Ah right, so it's more of a concern when defining generic fns (i.e. this lib) but you're saying there's nothing wrong with, say me having types in my own code of `Fix[Blah]`. Is that right?